server/weaverbird/steps/argmax.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from pandas import DataFrame
+from pydantic import Field
+
+from weaverbird.steps.base import BaseStep
+
+
+class ArgMaxStep(BaseStep):
+    name = Field('argmax', const=True)
+    column: str
```

Comment:
Tu peux importer ColumnName depuis weaverbird/types.py
```suggestion
    column: ColumnName
```
(au cas où on se met à supporter des noms de colonnes non-string)